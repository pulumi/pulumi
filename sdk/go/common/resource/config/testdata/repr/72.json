{"my:testKey":["0123456"]}