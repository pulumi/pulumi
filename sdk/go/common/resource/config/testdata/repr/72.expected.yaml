decrypt:
  my:testKey: '["0123456"]'
paths:
  my:testKey:
    value:
    - "0123456"
    string: '["0123456"]'
    redacted: '["0123456"]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAAF/+EFAABBnN0cmluZwwJAAcwMTIzNDU2
    secure: false
    isObject: true
  my:testKey[0]:
    value: "0123456"
    string: "0123456"
    redacted: "0123456"
    object: FBAABnN0cmluZwwJAAcwMTIzNDU2
    secure: false
    isObject: false
